### PR TITLE
Update example app

### PR DESCRIPTION
**Pull request by jnewc**
CHKT-1637

# What this PR does

The example app doesn't build currently. This fixes the build issues and gets the basic functionality of the app working again.

# Before merging

_QA_

- [ ] I manually tested this with a demo application (simulator)

### CODE REVIEW

File: Example/Sources/Network/Networking+Models.swift [hunk 1]
```diff
@@ -103,15 +103,15 @@ extension Networking {
                         ],
                         "PAYMENT_CARD": [
                             "networks": [
-                                "VISA": [
-                                    "surcharge": [
-                                        "amount": 109
-                                    ]
+                                "VISA": [:
+//                                    "surcharge": [
```
[the remark this answers is not in the document]
Comment (xevious78) [reply Example/Sources/Network/Networking+Models.swift:107]: Headless does not support surcharge at the moment!

File: Example/Sources/View Controller/MerchantAssetsViewController.swift [hunk 2]
```diff
@@ -9,54 +9,138 @@
 import PrimerSDK
 import UIKit
 
+fileprivate enum PrimerPaymentMethodTypeForAssets: String, CaseIterable, Equatable, Hashable {
```
Comment (jnewc) [inline Example/Sources/View Controller/MerchantAssetsViewController.swift:12]: This is to allow iteration over the various assets as this type is internal for the SDK.

File: Example/Sources/Network/Networking+Models.swift (hunk 1 above)
[the remark this answers is not in the document]
Comment (jnewc) [reply Example/Sources/Network/Networking+Models.swift:107]: Well that explains it!

**Review by borisprimer — APPROVED** [review]
Looks fine to me!

**Review by NQuinn27 — APPROVED** [review]
LGTM

**Pull request merged**